src/main/resources/default/assets/wondergem/font-awesome/css/font-awesome.css

```diff
@@ -6,8 +6,8 @@
  * -------------------------- */
 @font-face {
   font-family: 'FontAwesome';
-  src: url('/assets/wondergem/font-awesome/fonts/fontawesome-webfont.eot?v=4.5.0');
-  src: url('/assets/wondergem/font-awesome/fonts/fontawesome-webfont.eot?#iefix&v=4.5.0') format('embedded-opentype'), url('/assets/wondergem/font-awesome/fonts/fontawesome-webfont.woff2?v=4.5.0') format('woff2'), url('/assets/wondergem/font-awesome/fonts/fontawesome-webfont.woff?v=4.5.0') format('woff'), url('/assets/wondergem/font-awesome/fonts/fontawesome-webfont.ttf?v=4.5.0') format('truetype'), url('/assets/wondergem/font-awesome/fonts/fontawesome-webfont.svg?v=4.5.0#fontawesomeregular') format('svg');
```

Comment:
are we 100% sure that killing off the "iefix" doesn't harm anything?

Reply:
This is only relevant for some legacy browsers (IE6 - IE8):
https://stackoverflow.com/a/8636675/9758089